src/users/users.controller.ts

```
@@ -1,14 +1,16 @@
-import { CreateUserDto } from './dto/create-user.dto';
+// import { CreateUserDto } from './dto/create-user.dto'; // DTO is no longer used
 
 @Controller('users')
 export class UsersController {
   constructor(private readonly usersService: UsersService) {}
+  private readonly logger = new Logger(UsersController.name);
 
   @Post()
-  create(@Body() createUserDto: CreateUserDto) {
-    return this.usersService.create(createUserDto);
+  create(@Body() userData: any) { // Bypassing DTO validation by using `any`
```

ESLint

[error] Insert `⏎···`

(prettier/prettier)